catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/metacard/workspace/WorkspaceTransformer.java

```diff
@@ -178,9 +178,9 @@ public Metacard transform(Map<String, Object> map) {
     }
 
     public String toMetacardXml(Metacard m) {
-        try {
-            return IOUtils.toString(catalogFramework.transform(m, "xml", null)
-                    .getInputStream());
+        try (InputStream stream = catalogFramework.transform(m, "xml", null)
+                .getInputStream()) {
+            return IOUtils.toString(stream);
         } catch (Exception e) {
```

Comment:
I don't think we should be catching `Exception`, and I'm not sure returning a blank string makes sense here.